source/javascripts/site.js

```diff
@@ -1,15 +1,6 @@
-
 // Expands About section on homepage
-// function showHide() {
-//     let about = document.getElementById("about");
-//     if (about.style.display === "none") {
-//       about.style.display = "grid";
-//     } else {
-//       about.style.display = "none";
-//     }
-//   }
 
-  function showHide() {
-    let about = document.getElementById("about");
-    about.classList.toggle("slide-open")
-  }
\ No newline at end of file
+function showHide() {
+  let about = document.getElementById("about");
+  about.classList.toggle("slide")
```

Comment:
Just changed the name to `slide` from `slide-open`, since the class applies for both opening and closing

Reply:
perfect